tests/functional/aws-node-sdk/test/bucket/getCorsResponse.js

```diff
@@ -0,0 +1,1083 @@
+                'Access-Control-Request-Headers': 'Origin, Accept, ' +
+                'Content-Type',
+            };
+            methodRequest({ method: 'OPTIONS', headers, code: 'AccessForbidden',
+            headersResponse: null }, done);
+        });
+        it('send OPTIONS request with not allowed origin', done => {
+            const headers = {
+                'Origin': 'http://www.forbiddenwebsite.com',
+                'Access-Control-Request-Method': 'GET',
+            };
+            methodRequest({ method: 'OPTIONS', headers, code: 'AccessForbidden',
+            headersResponse: null }, done);
+        });
+        it('send OPTIONS request with listed method "POST"', done => {
+            const headers = {
+                'Origin': 'http://www.allowedwebsite.com',
+                'Access-Control-Request-Method': 'POST',
+            };
+            const headersResponse = {
+                'access-control-allow-origin': 'http://www.allowedwebsite.com',
+                'access-control-allow-methods': 'PUT, POST, DELETE, GET',
+                'access-control-allow-credentials': 'true',
+                'vary': 'Origin, Access-Control-Request-Headers, ' +
+                    'Access-Control-Request-Method',
+            };
+            methodRequest({ method: 'OPTIONS', headers, code: 200,
+            headersResponse }, done);
+        });
+        it('send OPTIONS request with listed method "PUT"', done => {
+            const headers = {
+                'Origin': 'http://www.allowedwebsite.com',
+                'Access-Control-Request-Method': 'PUT',
+            };
+            const headersResponse = {
+                'access-control-allow-origin': 'http://www.allowedwebsite.com',
+                'access-control-allow-methods': 'PUT, POST, DELETE, GET',
+                'access-control-allow-credentials': 'true',
+                'vary': 'Origin, Access-Control-Request-Headers, ' +
+                    'Access-Control-Request-Method',
+            };
+            methodRequest({ method: 'OPTIONS', headers, code: 200,
+            headersResponse }, done);
+        });
+        it('send OPTIONS request with listed method "DELETE"', done => {
+            const headers = {
+                'Origin': 'http://www.allowedwebsite.com',
+                'Access-Control-Request-Method': 'DELETE',
+            };
+            const headersResponse = {
+                'access-control-allow-origin': 'http://www.allowedwebsite.com',
+                'access-control-allow-methods': 'PUT, POST, DELETE, GET',
+                'access-control-allow-credentials': 'true',
+                'vary': 'Origin, Access-Control-Request-Headers, ' +
+                    'Access-Control-Request-Method',
+            };
+            methodRequest({ method: 'OPTIONS', headers, code: 200,
+            headersResponse }, done);
+        });
+    });
+
+    describe('CORS allows method GET and allows one origin', () => {
```

Comment:
is this different from the tests below only because of the allowed method in the cors config?  please clarify